tools/validators/instance_validator/validate/handler.py

```diff
@@ -164,9 +167,10 @@ def _ParseArgs(self, args):
 
     parser.add_argument(
         '-i', '--input',
-        dest='filename',
+        action='append',
+        dest='filenames',
         required=True,
-        help='Filepath to YAML building configuration',
+        help='Filepaths to YAML building configurations',
```

Comment:
nit: is it just one filepath which contains various config fileS?

Reply:
No, I enabled the 'append' option for the parameter, so the user can run it like: python instance_validator.py -i file1 -i file2

Even if only one input file is provided, the parameter will still be resolved to a list with one element.

Thanks for reminding me to update the README.

I wonder if we should add another --input-dir parameter that is mutually exclusive with --input which behaves in the way you were thinking.

Reply:
I wonder if we need both options.
Providing a directory which picks up all the files might be the simplest?

Just thinking about maintainability and keeping it simple for the user. 
What do you think?

Reply:
.

Reply:
I think I like having the option for both individual file validation and validation of an entire directory. It should make certain test / development tasks easier if we allow both (e.g. if the user has a particular file they're trying to fix and doesn't want to run the validator on the whole directory as they're iterating on it).

I opened a bug to add the new --input-dir parameter.